ci-operator/config/jboss-fuse/camel-k-test-container/jboss-fuse-camel-k-test-container-main__camel-k-ocp4.14-lp-rosa.yaml

```diff
@@ -0,0 +1,58 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: camel-k-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: camel-k-interop-aws-rosa-sts
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 1}},
+          {"step": "rosa-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 1}},
+          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI", "group": {"name": "operator", "priority": 1}},
+          {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}},
+          {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
+          {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}},
+          {"step": "camel-k-interop-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "CMLK", "group": {"name": "operator", "priority": 2}}
+        ]
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPTOCPCI
+      FIREWATCH_JIRA_SERVER: https://issues.redhat.com
+      OPERATORS: |
+        [
+            {"name": "red-hat-camel-k", "source": "redhat-operators", "channel": "latest", "install_namespace": "openshift-operators"}
+        ]
+    test:
+    - ref: install-operators
+    - ref: camel-k-interop-tests
```

Comment:
Following what we did for the rosa-hypershift testing we may want to include a cluster health check after the operator is installed.

https://steps.ci.openshift.org/reference/cucushift-installer-check-cluster-health

This is provided by the openshift team and is similar to the hypershift health check that we use here: https://github.com/openshift/release/pull/45666/files#diff-a5dd822947a977dd0b37e0d81e96710e0e6c4204f62607061a5faebfd7605b06R52

I'm curious to get your thoughts on including a health check before we run the PQE test suite.
- For rosa hypershift we initially used a health check after the operator install because we weren't ready to run PQE test suites yet, so now we have to decide if we should stop using these health checks or keep using them with the PQE test suites.